src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/syntaxes/aspnetcorerazor.tmLanguage.json

```diff
@@ -556,7 +556,7 @@
       "contentName": "source.cs",
       "patterns": [
         {
-          "include": "source.cs"
+          "include": "source.cs#class-or-struct-members"
```

Comment:
This is the fix for the nullable issue, specifically telling TextMate that the contents of `@code` and `@functions` blocks are class members.

At least thats my intepretation of what this does :)